src/ui/validation.yml

```diff
@@ -7,3 +7,6 @@ urlInvalid:
 urlInsecure:
   en: Please specify a URL with a secure protocol (HTTPS)
   de: Bitte geben Sie eine Adresse mit sicherem Protokoll (HTTPS) an
+markdownBoldOrItalicWithinHeaders:
+  en: Bold or italic should not be used within headers
```

Comment:
I think this should read:
"Bold or italic may only be used to emphasise parts of the header", we actually allow that.